db/schema.rb

```diff
@@ -32,9 +32,6 @@
     t.float "depth", limit: 24
     t.bigint "pipeline_run_id"
     t.string "drug_family"
-    t.integer "level"
-    t.float "drug_gene_coverage", limit: 24
```

Comment:
are these supposed to go away?

Reply:
it's not in the migrations. i think it'sok.